build-system/tasks/performance/index.js

```diff
@@ -32,13 +33,17 @@ async function performance() {
   await compileScripts(urls);
   await rewriteScriptTags(urls);
   await getMetrics(urls, {headless, runs});
+  runTests();
   printReport(urls);
 }
 
-performance.description = '  Runs web performance test on current branch';
+performance.description = 'Runs web performance test on current branch';
 
 performance.flags = {
-  'nobuild': 'Does not compile javascripts before running tests',
+  'nobuild': '  Does not compile javascripts before running tests',
```

Comment:
nit: While you're here, can you re-word "Does not compile javascripts..." to "Does not compile minified runtime..."?

Reply:
Done